blsSignatures/blsSignatures.go

```diff
@@ -240,19 +256,26 @@ func PublicKeyFromBytes(in []byte, trustedSource bool) (PublicKey, error) {
 }
 
 func PrivateKeyToBytes(priv PrivateKey) []byte {
-	return bls12381.NewFr().Set(priv).ToBytes()
+	bytes := new(fr.Element).Set(priv).Bytes()
```

Comment:
The Bytes method seems not to mutate, so I don't think we need the copy here

Reply:
Done.